RNA-seq/03-gastric_cancer_exploratory.Rmd

```diff
@@ -128,7 +128,7 @@ ggplot2::ggsave(pca_plot_file, plot = ggplot2::last_plot())
 We don't have batch information (i.e., when the samples were run) for this 
 particular experiment, but let's imagine that `SRR585574` and `SRR585576` were
 run separately from all other samples.
-We'll put this in new "toy" column in `gene_summarized`.
+We'll add this as a new "toy" column in the sample data (`colData`) for `gene_summarized`.
```

Comment:
okay now that we are adding this to vst_data, let's make this slightly more general.
```suggestion
We'll add this as a new "toy" column in the sample data (`colData`).
```